### PR TITLE
Add temporary environment variable for disabling NFS.

### PR DESCRIPTION
**Pull request by oliverchang**
NFS keeps causing reliability and scalability issues with our scale.

This environment variable is to enable easy testing/reverting of this.
If this works out, the supporting code will be removed too.

### CODE REVIEW

**Comment by oliverchang** [comment]
TBR

File: src/clusterfuzz/_internal/google_cloud_utils/storage.py [hunk 1]
```diff
@@ -931,7 +931,9 @@ def get_file_size_from_cache_if_exists(file_path):
 
 def get_cache_file_path(file_path):
   """Return cache file path given a local file path."""
-  if not environment.get_value('NFS_ROOT'):
+  # TODO(ochang): Completely remove NFS support.
+  if (not environment.get_value('NFS_ROOT') or
+      environment.get_value('DISABLE_NFS')):
```
Comment (Alan32Liu) [inline src/clusterfuzz/_internal/google_cloud_utils/storage.py:936]: Do we need the outermost parentheses?

**Review by Alan32Liu — APPROVED** [review]

File: src/clusterfuzz/_internal/google_cloud_utils/storage.py (hunk 1 above)
Comment (oliverchang) [reply src/clusterfuzz/_internal/google_cloud_utils/storage.py:936]: it's for line continuation (see https://google.github.io/styleguide/pyguide.html#32-line-length)

Comment (Alan32Liu) [reply src/clusterfuzz/_internal/google_cloud_utils/storage.py:936]: Thanks!

**Comment by Alan32Liu** [comment]
> gcbrun

A silly question: What does this do and why is it needed?
I got an error in my `OSS-FUZZ` PR saying 'Needs /gcbrun'.

**Comment by jonathanmetzman** [comment]
gcbrun kicks off google cloud build.
on clusterfuzz it's simple to use, nothing else is needed.
In OSS-Fuzz, it's more complicated. It's for doing trial builds. See this doc for details: https://docs.google.com/document/d/1kL8Rdl9aOHsBvsY2kn4Ti00NSEEIclw6gou-pHcyOV8/edit

**Pull request merged**